### PR TITLE
[CARBONDATA-1792] Add example of data management for Spark2.X

### PR DESCRIPTION
**Pull request by Xaprice**
[CARBONDATA-1792]add example of data management for Spark2.X

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by chenliang613** [comment]
add to whitelist

**Comment by chenliang613** [comment]
Please follow the below template to provide pull request description.
--------------------------------------------------------------------
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

**Comment by chenliang613** [comment]
retest this please

**Comment by chenliang613** [comment]
the pr title please change to : [CARBONDATA-1792] Add example of data management for Spark2.X

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1362/

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: examples/spark2/src/main/scala/org/apache/carbondata/examples/DataManagementExample.scala [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+         | country String,
+         | name String,
+         | phonetype String,
+         | serialname String,
+         | salary Int,
+         | floatField float
+         | ) STORED BY 'carbondata'
+       """.stripMargin)
+
+    // data.csv has 10 lines
```
Comment (chenliang613) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/DataManagementExample.scala]: this row comment is not required : 

// data.csv has 10 lines

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1383/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1831/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**